dagshub/upload/wrapper.py

```diff
@@ -153,7 +154,7 @@ def _log_upload_details(self, data, res, files):
             logger.debug(f"Response ({res.status_code})\n")
 
         if res.status_code == 200:
-            logger.info("Upload finished successfully!")
+            logger.warning("Upload finished successfully!")
```

Comment:
And here?
@guysmoilov  I saw your comment but I don't think messages like these should be reported as warnings.